### PR TITLE
Update Test4.txt

### PR DESCRIPTION
**Pull request by fomotuyi**
It is updated

### CODE REVIEW

File: Test4.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 I will like to be good
 You will learn Git and Git command
+kk
```
Comment (deandebola) [inline Test4.txt:3]: This is good

**Comment by fomotuyi** [comment]
kk

**Comment by fomotuyi** [comment]
kk

**Review by deandebola — APPROVED** [review]
Approved

**Pull request merged**